apps/services/auth/delegation-api/src/app/delegations/delegations.controller.ts

```diff
@@ -47,10 +54,11 @@ export class DelegationsController {
         },
       },
       query: {
-        scope: {
+        scopes: {
```

Comment:
I guess we don't need to version this or make it backwards compatible? As there are no outside clients (?).